src/gui/widgets/MixerLine.cpp

```diff
@@ -262,21 +282,44 @@ void MixerLine::contextMenuEvent( QContextMenuEvent * )
 	contextMenu->addSeparator();
 
 	QMenu colorMenu(tr("Color"), this);
-	colorMenu.setIcon(embed::getIconPixmap("colorize"));
-	colorMenu.addAction(tr("Change"), this, SLOT(selectColor()));
-	colorMenu.addAction(tr("Reset"), this, SLOT(resetColor()));
-	colorMenu.addAction(tr("Pick random"), this, SLOT(randomizeColor()));
-	contextMenu->addMenu(&colorMenu);
+	if (!autoTrackLink)
+	{
+		colorMenu.setIcon(embed::getIconPixmap("colorize"));
+		colorMenu.addAction(tr("Change"), this, SLOT(selectColor()));
```

Comment:
I find ``SLOT()`` to be redundant mainly because any function can act as a slot now. If you want to be consistent with the current codebase at the moment, then I would say it is not that big of an issue.

https://doc.qt.io/qt-5/signalsandslots.html#:~:text=You%20can%20also%20connect%20to%20functors%20or%20C%2B%2B11%20lambdas%3A

Reply:
Yes I know now but I just thought it is more consistent with what is already there.